src/mondeath.cpp

```diff
@@ -723,3 +726,28 @@ void make_mon_corpse(monster *z, int damageLvl)
     }
     g->m.add_item_or_charges(z->pos(), corpse);
 }
+
+void mdeath::preg_roach(monster *z)
+{
+    int num_roach=rng(1,3);
```

Comment:
Style nit for future reference (I'll just fix it up while merging if there are no problems in the code), we like spaces around operators, makes things a bit more readable.